.swiftlint.yml

```diff
@@ -35,7 +35,6 @@ only_rules:
   - force_try
   - force_unwrapping
   - function_body_length
-  - function_parameter_count
```

Comment:
I disabled this rule cause it only allows 5 parameters per method, which doesn't make sense IMO.